src/Cmts.ml

```diff
@@ -462,12 +476,16 @@ let relocate t ~src ~before ~after =
     if Conf.debug then
       Format.eprintf "relocate %a to %a and %a@\n%!" Location.fmt src
         Location.fmt before Location.fmt after ;
+    let append_and_sort x acc =
+      List.rev_append x acc
+      |> List.sort ~compare:(Comparable.lift Location.compare_start ~f:snd)
+    in
     update_multi t.cmts_before src before ~f:(fun src_cmts dst_cmts ->
-        List.append src_cmts dst_cmts) ;
+        append_and_sort src_cmts dst_cmts) ;
```

Comment:
Arguments flipped ?

Reply:
I don't follow you.. can you explain ?

Reply:
The 2 next occurrences are `append_and_sort dst_cmts src_cmts`. But this should not be an issue because of the sort.

Reply:
I've cleaned the code a bit.